components/src/preact/components/error-display.tsx

```diff
@@ -10,11 +14,36 @@ export class UserFacingError extends Error {
 export const ErrorDisplay: FunctionComponent<{ error: Error }> = ({ error }) => {
     console.error(error);
 
+    const ref = useRef<HTMLDialogElement>(null);
+
     return (
         <div className='h-full w-full rounded-md border-2 border-gray-100 p-2 flex items-center justify-center flex-col'>
             <div className='text-red-700 font-bold'>Error</div>
-            <div>Oops! Something went wrong.</div>
-            {error instanceof UserFacingError && <div className='text-sm text-gray-600'>{error.message}</div>}
+            <div>
+                Oops! Something went wrong.
+                {error instanceof UserFacingError && (
+                    <>
+                        {' '}
+                        <button className='text-sm text-gray-600' onClick={() => ref.current?.showModal()}>
```

Comment:
```suggestion
                        <button className='text-sm text-gray-600 hover:text-gray-300' onClick={() => ref.current?.showModal()}>
```
So you see more clearly that you can click this.